### ISSUE TITLE
Allure annotaions is not passed to report.

### ISSUE THREAD

**Issue by jalona** [allure-framework/allure-cucumberjvm#27]
There are questions that I would like to have answers. Please help me
 
1) Realization `@BeforeStep` `@AfterStep` when we can expect?
2) To test Cucubmer Annotated `@Issue @Severity @Description` not fall into the report allure. What is the reason?
Example:

```
@Issue("JIRA-123")
@И("открыто окно идентификации клиента")
    public void openIdentForm() throws Throwable {
        driver.findElement(LocatorElements.processAproveed).click();
        driver.findElement(LocatorElements.identificationClient).click();
    }
```

**Comment by clicman** [comment]
Looks like you found the bug :)

**Comment by jalona** [comment]
When for the time expect fixing this bug ?

**Comment by clicman** [comment]
As soon as I`ll get some free time. 
Another way to get it fixed - is to send PR.

**Comment by hemanthsridhar** [comment]
@clicman  Hi. Any update on this?

**Comment by thelazytester** [comment]
To use Severity, Issue and TestCaseId you have to add those annotation in the .feature.
Example:

> Feature: As agent I want to log out of the webapplication so that i can end the session.
>   Background: 
>     Given Agent is on webapp
>     When Agent enters Username as "root" and Password "root"
>     And Agent clicks on Login button
> 
>   **@SeverityLevel.CRITICAL @Issue("JIRA-1) @TestCaseId("JIRA-642")**
>   Scenario: Verify Logout Functionality
>     Given Agent is on HomePage
>     When Agent clicks on Logout button
>     Then Agent should get logged out

It worked for me.

**Comment by hemanthsridhar** [comment]
@ankit-ojha  Whatever you have mentioned is allure cucumber plugin developed by Anton.  This is not allure cucumber plugin, this is allure cucumber jvm adaptor.

**Comment by clicman** [comment]
Fixed in master. Will available in 1.6 release
